Marlin/Marlin_main.cpp

```diff
@@ -4145,11 +4146,21 @@ inline void gcode_M105() {
     }
   #endif
 
+  // Get error codes from present cartridges
```

Comment:
This is commented out because it works, but can cause printers without upgraded cartridges to hang. We're going to wait until all cartridges have been upgraded before implementing